web-api/src/personalization.rs

```diff
@@ -56,11 +57,11 @@ pub struct ConfigExtension {
 
     #[as_ref]
     #[serde(default)]
-    pub(crate) personalization: PersonalizationConfig,
+    pub(crate) personalization: Config,
 }
 
 #[derive(Debug, Deserialize, Serialize)]
-pub(crate) struct PersonalizationConfig {
```

Comment:
Don't rename if not needed to reduce the changes to only the ones that are needed for this PR.